python/benchmark/databricks/init-pip-cuda-11.8.sh

```diff
@@ -2,10 +2,10 @@
 # set portion of path below after /dbfs/ to dbfs zip file location
 SPARK_RAPIDS_ML_ZIP=/dbfs/path/to/spark-rapids-ml.zip
 BENCHMARK_ZIP=/dbfs/path/to/benchmark.zip
-# IMPORTANT: specify rapids fully 23.4.0 and not 23.4
-# also RAPIDS_VERSION (python) fields should omit any leading 0 in month/minor field (i.e. 23.4.0 and not 23.04.0)
-# while SPARK_RAPIDS_VERSION (jar) should have leading 0 in month/minor (e.g. 23.04.0 and not 23.4.0)
-RAPIDS_VERSION=23.4.0
+# IMPORTANT: specify rapids fully 23.6.0 and not 23.6
+# also RAPIDS_VERSION (python) fields should omit any leading 0 in month/minor field (i.e. 23.6.0 and not 23.06.0)
+# while SPARK_RAPIDS_VERSION (jar) should have leading 0 in month/minor (e.g. 23.06.0 and not 23.6.0)
+RAPIDS_VERSION=23.6.0
 SPARK_RAPIDS_VERSION=23.04.0
```

Comment:
Should SPARK_RAPIDS_VERSION be updated as well?

Reply:
Yes it should.   My diff shows it is updated though.